index.js

```diff
@@ -0,0 +1,31 @@
+let theBooks = [];
+
+const addButton = document.querySelector('.button');
+addButton.addEventListener('click', () => {
+  const bookContainer = document.createElement('div');
+  const titleBook = document.createElement('h5');
+  const authorBook = document.createElement('h5');
+  const removeButton = document.createElement('button');
+  const breakLine = document.createElement('hr');
+  const inputTitle = document.querySelector('.title');
+  const inputAuthor = document.querySelector('.author');
+  titleBook.innerHTML = inputTitle.value;
+  authorBook.innerHTML = inputAuthor.value;
+  removeButton.innerHTML = 'Remove';
+
+  bookContainer.append(titleBook, authorBook, removeButton, breakLine);
+  const bookList = document.querySelector('.book-list');
+  bookList.append(bookContainer);
+
+  theBooks.push({ title: inputTitle.value, author: inputAuthor.value });
+
+  window.localStorage.setItem('booksArray', JSON.stringify(theBooks));
```

Comment:
- [ ] Please make sure that data persisted to the localStorage is getting loaded when the page loads. At the moment, if you add and refresh then it disappears in the UI but is still in the `localStorage`. You can create a function `displayBooks` and call it this way
```
  document.addEventListener('DOMContentLoaded', displayBooks);
```